### PR TITLE
Instance API improved to support both old and new instance panel

### PR DESCRIPTION
**Pull request by SergeyKemaev**
Support of API changes made in https://github.com/qubell/vermilion/pull/5079

Impact:
- instance-related tests

### CODE REVIEW

**Comment by SergeyKemaev** [comment]
@vasichkin please review (mainly check that old API has not been broken)

File: qubell/api/private/instance.py [hunk 1]
```diff
@@ -58,16 +58,17 @@ def environment(self):
 
     @lazyproperty
     def environments(self):
-        list_environments_json = lambda: self.json()['environments']
-        return EnvironmentList(list_json_method=list_environments_json, organization=self).init_router(self._router)
+        return self.organization.environments
```
Comment (vasichkin) [inline qubell/api/private/instance.py]: I'd prefer to get this information from instance.json, if it is still there

File: qubell/api/private/instance.py [hunk 2]
```diff
@@ -58,16 +58,17 @@ def environment(self):
...
 
     @lazyproperty
     def applicationId(self):
-        return self.json()['applicationId']
+        j = self.json()
+        return j.get('applicationId', j.get('application', {}).get('id'))
```
Comment (vasichkin) [inline qubell/api/private/instance.py]: I hope this works as should

File: qubell/api/private/instance.py [hunk 3]
```diff
@@ -77,6 +78,11 @@ def submodules(self):
             return InstanceList(list_json_method=lambda: self.json()['components'], organization=self.organization).init_router(self._router)
         return InstanceList(list_json_method=lambda: self.json()['submodules'], organization=self.organization).init_router(self._router)
 
+    @lazyproperty
+    def destroyAt(self):
+        j = self.json()
+        return j.get('destroyAt', False)
```
Comment (vasichkin) [inline qubell/api/private/instance.py]: Why do we need this?

**Comment by vasichkin** [comment]
I'd like comments for those places, where old and new logic meets. So, one time we could easily clean old code.

File: qubell/api/private/instance.py (hunk 1 above)
Comment (SergeyKemaev) [inline qubell/api/private/instance.py]: Unfortunately no, there is no `environments` field in instance.json anymore.

File: qubell/api/private/instance.py (hunk 3 above)
Comment (SergeyKemaev) [inline qubell/api/private/instance.py]: `destroyAt` is an Option in instance viewmodel on server. Thus when there is no destroy time, there will be no field `destroyAt` in json and call like `self.json()['destroyAt']` throws KerError

**Comment by SergeyKemaev** [comment]
Code commented and slightly reworked to make it more concise

**Comment by vasichkin** [comment]
Cool, could be merged when needed

**Pull request merged**